src/app/utils/tests/ArrayUtils.spec.ts

```diff
@@ -21,3 +23,38 @@ describe('ArrayUtils', () => {
         });
     });
 });
+
+describe('Table2DWithPossibleNegativeIndices', () => {
+    it('should return empty when accessing a non existing element', () => {
+        // Given a table
+        const table: Table2DWithPossibleNegativeIndices<number> = new Table2DWithPossibleNegativeIndices();
+        // When getting an element that does not exist
+        const element: MGPOptional<number> = table.get(new Coord(0, 0));
+        // Then it should be empty
+        expect(element.isAbsent()).toBeTrue();
+    });
+    it('should return the accessed element after it has been set', () => {
+        // Given a table with a set element
+        const table: Table2DWithPossibleNegativeIndices<number> = new Table2DWithPossibleNegativeIndices();
+        table.set(new Coord(1, 2), 42);
```

Comment:
why some hardcoded value ? What's 42 the result of ? I don't understand ! Keep it simple please !

Reply:
j'ai maintenant nommé et explicité la dummy value !